cylc/flow/etc/tutorial/runtime-introduction/etc/python-job.settings

```diff
@@ -0,0 +1,12 @@
+# WARNING: This file contains an anti-pattern
```

Comment:
Any reason this one has not been made into a symlink?

Reply:
It didn't pass the tests when it was a symlink.

Reply:
Then it seems suspicious the others are passing

Reply:
The problems:
- The symlink points outside of the workflow source so gets broken on install
- There is inconsistency in the test scripts between the three:

  https://github.com/cylc/cylc-flow/blob/47ec638f85ee7a762c8a13736b204e64d2ec6330/cylc/flow/etc/tutorial/runtime-introduction/.validate#L18-L23

  https://github.com/cylc/cylc-flow/blob/9ad4c07f8daa6e339ed43ac388b45e53b0972b69/cylc/flow/etc/tutorial/consolidation-tutorial/.validate#L19-L21  

  (`runtime-tutorial` does not even have a `.validate` script)

Reply:
Yes. Might as well fix it in this PR, it's mission-creeped enow already.

Reply:
> I think the symlinks will need to be replaced by copies of the `python-job.settings`, because if the user hasn't yet installed `cylc-forecasting-workflow` and doesn't use `--no-run-name` when installing the workflows, the symlinks will be broken?

I don't think that this check is in scope, happy to check later because I'm fairly unhappy about what's going - but I'd like to check with @oliver-sanders  because he said that symlinks were ok - (No, I'm not convinced either.)

Reply:
Are the links not working in the other tutorials? Perhaps we have a relative/absolute issue, there are some `rsync` options which configure how `rsync` behaves when it finds links.

Might need to get rsync to `--copy-links`, can we do that with existing configuration?

Reply:
Note we have been using symlinks with the tutorials for years when they were in Rose, using `rsync` to install. Since `cylc install` is `rsync` underneath I would *expect* it to be the same deal?

Reply:
I'll have a look. I guess this is urgent - it ought to go in before the seminars

Reply:
This seems to have sorted itself out. I don't know what was going on, but it no longer appears to be going.

Reply:
Ah got it - `cylc get-resources tutorial` turns the symlinks into files